packages/examples/device_metrics/sensor_tag_ble_dell5k_mqtt_iotcc.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
```

Comment:
Okay will rename it.  For packages, we need to move device and metric entities separately.  This will result changes in many files.  In the interest of keeping no.of files changed for a PR less, I'll raise a separate one just for refactoring once this is merged.